src/Microsoft.Health.Fhir.Core/Configs/CorsConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.Core.Configs
+{
+    public class CorsConfiguration
+    {
+        public IList<string> Origins { get; } = new List<string>();
+
+        public IList<string> Headers { get; } = new List<string>();
+
+        public IList<string> Methods { get; } = new List<string>();
+
+        public int MaxAge { get; set; }
```

Comment:
Should default this to -1 which disables caching.

See - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age

Reply:
Turns out that .net restricts this to not be negative. I've updated this to be nullable, so that it can just not be specified.